upup/pkg/fi/cloudup/tests/bootstrapchannelbuilder/simple/manifest.yaml

```diff
@@ -25,6 +25,13 @@ spec:
     version: 1.14.5
   - id: k8s-1.8
     kubernetesVersion: '>=1.8.0'
+    manifest: coredns.addons.k8s.io/k8s-1.6.yaml
+    name: coredns.addons.k8s.io
```

Comment:
Surprised that we have both kube-dns and coredns here..  I would expect coredns not to appear unless you had the feature flag enabled, but if you did I would expect kube-dns not to appear.

Reply:
If we put the api values the way you recommend I think it will be cleaner.

Reply:
Meaning if we do the APIs we can have logic on which one to apply. If we do that I do not think we need a feature flag. But I am still confused how we do not have both deployed.